include/ledstripeffect.h

```diff
@@ -110,8 +110,8 @@ class LEDStripEffect : public IJSONSerializable
         int colorIndex = 0;
 
         auto componentsArray = src.as<JsonArrayConst>();
-        for (auto value : componentsArray)
-            colors[colorIndex++] = value.as<CRGB>();
+        for (auto v : componentsArray)

```

Comment:
Would a & eliminate a construct/copy here?

(It annoys me that 'auto' can't really be trusted here to do the right thing - it would be So Clean....)

Reply:
I'll go you one better with const auto & for the win!